cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -118,6 +118,16 @@ pub enum Inst {
     /// Materializes the requested condition code in the destination reg.
     Setcc { cc: CC, dst: Writable<Reg> },
 
+    /// Integer conditional move.
+    /// Overwrites the destination register.
+    Cmove {
+        /// Possible values are 2, 4 or 8. Checked in the related factory.
```

Comment:
Per comments I added to one of Chris's comments, I think we should generate 32-bit insns instead of 8- or 16-bit insns except where we absolutely can't avoid them (eg, stores), so as to avoid partial register stall hazards.

Reply:
Good point, I'll revisit this by re-checking all the lowering.